stl/inc/memory

```diff
@@ -3243,6 +3247,27 @@ public:
         return false;
     }
 
+    void wait(shared_ptr<_Ty> _Old, memory_order = memory_order::seq_cst) const noexcept {
+        for (;;) {
+            auto _Rep   = this->_Lock_and_load();
+            bool _Equal = this->_Ptr.load(memory_order::relaxed) == _Old._Ptr;
+            this->_Store_and_unlock(_Rep);
+            if (!_Equal) {
+                break;
+            }
+            __std_atomic_wait_direct(&this->_Ptr, &_Old._Ptr, sizeof(_Old._Ptr), _Atomic_wait_no_timeout);
+        }
+    }
+
+    void notify_one() noexcept {
+        this->_Ptr.notify_one();
+    }
+
+

```

Comment:
Nitpick: It seems unnecessary to have two newlines between `notify_one` and `notify_all`.